modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -29,14 +37,23 @@ def show
 
         private
 
-        def set_submission_attributes
-          @appeal ||= AppealsApi::NoticeOfDisagreement.find_by(id: params[:nod_id])
-          raise Common::Exceptions::RecordNotFound, params[:nod_id] unless @appeal
+        def set_notice_of_disagreement
+          @nod_id ||= params[:nod_id]
```

Comment:
I don't really see the need to hold onto both the nod_id and notice_of_disagreement as instance variables. Error if the params[:nod_id] is bad in some way before we can load the record, and then use that record for the ID everywhere else.

e.g.
```ruby
raise <snip>::ParameterMissing, 'nod_id' if params[:nod_id]&.blank?
@notice_of_disagreement = <snip>.find_by(id: params[:nod_id])
```